shell/platform/android/BUILD.gn

```diff
@@ -251,6 +251,18 @@ embedding_dependencies_jars =
                 ],
                 "list lines")
 
+action("check_imports") {
+  script = "//flutter/tools/android_illegal_imports.py"
+
+  sources = android_java_sources
+
+  # File does not actually get created, but GN expects us to have an output here.
+  outputs = [ "$root_out_dir/check_android_imports" ]
+
+  args = [ "--files" ] +
+         rebase_path(android_java_sources - [ "io/flutter/Log.java" ])
```

Comment:
nit: some scoping leak. Your thing is called check illegal imports but you're excluding one file. We should probably just exclude that file specifically for the android log check in the check itself rather than excluding it from all 'illegal imports'

Reply:
Changed this around so that the script itself does the check.